doc/articles/routed-events.md

```diff
@@ -0,0 +1,48 @@
+# Uno Support for _Routed Events_
+
+## Implemented Routed Events
+
+| Routed Event          | Android | iOS    | Wasm   |     |
+| --------------------- | ------- | ------ | ------ | --- |
+| _tapped events_
+| `Tapped `             | Yes     | Yes    | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.tapped) |
+| `DoubleTapped`        | Yes     | ??     | No     | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.doubletapped) |
+| _focus events_
+| `GotFocus`            | Yes     | Yes(1) | Yes(1) | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.gotfocus) |
+| `LostFocus`           | Yes     | Yes(1) | Yes(1) | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.lostfocus) |
+| _keyboard events_
+| `KeyDown`             | No      | Yes(2) | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.keydown) |
+| `KeyUp`               | No      | Yes(2) | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.keyup) |
+| _pointerd events_
+| `PointerCanceled`     | Yes     | No     | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.pointercanceled) |
+| `PointerCaptureLost`  | Yes     | No     | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.pointercapturelost) |
+| `PointerEntered`      | Yes     | No     | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.pointerentered) |
+| `PointerExited`       | Yes     | No     | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.pointerexited) |
+| `PointerMoved`        | Yes     | No     | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.pointermoved) |
+| `PointerPressed`      | Yes     | No     | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.pointerpressed) |
+| `PointerReleased`     | Yes     | No     | Yes    | [Documentation](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.pointerreleased) |
+
+Notes:
+
+1. **Focus** events::
+   * iOS: The concept of _focus_ is emulated because not supported by the platform.
```

Comment:
> The concept of _focus_ is emulated because not supported by the platform.

Does this have any practical implications? This could apply to large parts of Uno, I don't think it necessarily needs to be mentioned.

Reply:
Is it documented elsewhere?